lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParser.h

```diff
@@ -58,7 +58,7 @@ class DWARFASTParser {
   virtual void EnsureAllDIEsInDeclContextHaveBeenParsed(
       CompilerDeclContext decl_context) = 0;
 
-  virtual std::string GetDIEClassTemplateParams(const DWARFDIE &die) = 0;
+  virtual std::string GetDIEClassTemplateParams(DWARFDIE die) = 0;
```

Comment:
I could have handled this in another way, but I think this type is small enough to be passed by value.